joomla/joomla-cms
# [4.1] Use emojis vs google fonts for the accessibility plugin

Pull Request for Issue #36928

### Summary of Changes

Set "useEmojis" as recommended by the docs: https://ranbuch.github.io/accessibility/#icon-image

### Testing Instructions

Apply this PR and make sure no google fonts are loaded when using the accessibility plugin

### Actual result BEFORE applying this Pull Request

Before this change the default value is used which uses google fonts.

### Expected result AFTER applying this Pull Request

We no longer use google fonts but emojis.

### Documentation Changes Required

none
